syncstorage/tests/test_wsgiapp.py

```diff
@@ -148,6 +139,31 @@ def new_do_request(req, *args, **kwds):
         else:
             assert False, "timer metrics were not emitted"
 
+    def test_503s_for_migrating_users(self):
+        user = {
+            "uid": 42,
+            "fxa_uid": "foobar",
+            "fxa_kid": "001122",
+            "hashed_fxa_uid": "aabbcc",
+        }
+        app = self._make_test_app(user)
+        req = self._make_signed_req(user["uid"], user)
+
+        # There's no public API for marking a user as migrated.
+        storage = get_storage(req)
+        with storage.dbconnector.connect() as connection:
+            connection.execute("""
+                INSERT INTO migration (fxa_uid, started_at)
+                VALUES (:fxa_uid, :started_at)
+                /* queryName=migrate */
```

Comment:
nit: may want to add a `state` value here for completeness (I guess the test works because it defaults to `NULL` so it's not strictly necessary, but you use the string "in_progress" for a different test above).